src/lib/api.ts

```diff
@@ -1046,6 +1046,41 @@ export const searchParentContent = cache(
     },
 );
 
+/**
+ * Search content in a Site or specific SiteSpaces.
+ */
+export const searchSiteContent = cache(
+    'api.searchSiteContent',
+    async (
+        organizationId: string,
+        siteId: string,
+        query: string,
+        siteSpaceIds: string[] = [],
+        /** A cache bust key to avoid revalidating lot of cache entries by tags */
+        cacheKey?: string,
+        options?: CacheFunctionOptions,
+    ) => {
+        const response = await api().orgs.searchSiteContent(
+            organizationId,
+            siteId,
+            {
+                query,
+                siteSpaceIds,
+            },
+            undefined,
+            {
+                ...noCacheFetchOptions,
+                signal: options?.signal,
+            },
+        );
+
+        return cacheResponse(response, {
+            ttl: 60 * 60,
```

Comment:
1h seems a lot for search results. We should make it shorter, like 5 minutes top.

Reply:
There is the cache bust now, so 1h is fine, 5min will not even work with KV

Reply:
If it's covered by the cache bust then yeah.